SUPPORT.md

```diff
@@ -0,0 +1,5 @@
+# Ark Support
```

Comment:
follow up thought: is `SUPPORT.md` a sort of best practice file to always include, similar to `CONTRIBUTING.md`? if not this could actually just incorporated INTO the main readme